node/server/server.ts

```diff
@@ -141,6 +165,7 @@ const serve = async ({
   onAfterDownload,
   onBeforeDownload,
   port,
+  servePath,
```

Comment:
The new `servePath` will be used to write any temporary files, such as the stage 2 file, rather than using a temporary directory.

While working on this, I noticed that we weren't previously cleaning up the temporary directories we created. We address that here.